jobs/pr_gate/docker/monorail/config.json

```diff
@@ -0,0 +1,64 @@
+{
+    "CIDRNet": "172.31.128.0/22",
+    "amqp": "amqp://localhost",
+    "apiServerAddress": "172.31.128.250",
```

Comment:
after talking with @PengTian0  & @changev , That's because the Jenkins slave NIC ```em1``` are all with IP ```172.31.128.250```( which was set by @keedya ). because docker are using host mode network so the config.json should align ```em1```'s IP.
this should be noted in README.md, because it's tricky.